core/src/plugins/exec/run.ts

```diff
@@ -32,8 +32,34 @@ export type ExecRun = GardenSdkActionDefinitionActionType<typeof execRun>
 execRun.addHandler("run", async ({ artifactsPath, log, action, ctx }) => {
   const startedAt = new Date()
   const { command, env, artifacts } = action.getSpec()
+  const runCommandErrors: unknown[] = []
```

Comment:
If there's always one element in the array, I think it's cleaner to use `let` here:
```suggestion
  let runCommandError: unknown | undefined
```

Reply:
`runCommandErrors` is never reassigned, it will give you a type error if you use `let`.

Reply:
it's never reassigned because you push an element to the array, but why use an array in the first place if it'll never have more than one element?

Reply:
aaah, that's what you meant. Sorry didn't understand you from your first comment. Sure thing, that makes sense.

Reply:
@10ko I've pushed a commit with the change I had in mind, does it look good to you?

Reply:
No worries :D